var/spack/repos/builtin/packages/axl/package.py

```diff
@@ -36,6 +37,9 @@ class Axl(CMakePackage):
     depends_on('kvtree')
     depends_on('zlib', type='link')
 
+    depends_on('kvtree@main', when='@main')
+    depends_on('kvtree@1.3.0', when='@0.6.0')
+
```

Comment:
Will Spack users need access to the new `ENABLE_PTHREADS` cmake option, or is it not necessary here?